engine/runtime/src/main/resources/Builtins.enso

```diff
@@ -243,6 +243,16 @@ type Error
     to_text : Text
     to_text = @Builtin_Method "Error.to_text"
 
+@Builtin_Type
+type Prim_Warning
+    type Prim_Warning
+
+    attach : Any -> Any -> Any
+    attach value warning = @Builtin_Method "Prim_Warning.attach"
+
+    get_all : Any -> Vector.Vector
+    get_all value = @Builtin_Method "Prim_Warning.get_all"
```

Comment:
There are a few methods missing from this list `get_origin` and `get_reassignments`.